internal/version/version.go

```diff
@@ -4,24 +4,106 @@
 package version
 
 import (
+	"context"
 	"encoding/json"
+	"errors"
+	"fmt"
 	"io/ioutil"
 	"net/http"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"go.uber.org/zap"
+
+	"gopkg.in/spacemonkeygo/monkit.v2"
+)
+
+var (
+	mon = monkit.Package()
+	// Timestamp is the UTC timestamp of the compilation time
+	Timestamp string
+	// CommitHash is the git hash of the code being compiled
+	CommitHash string
+	// Version is the semantic version set at compilation
+	// if not a valid semantic version Release should be false
+	Version = "0.1.0"
+	// Release indicates whether the binary compiled is a release candidate
+	Release bool
+	// Build is a struct containing all relevant build information associated with the binary
+	Build V
+	// Allowed ensures, the client is still on the allowed versions returned by the control server
+	Allowed bool
 )
 
-// V contains all necessary values for comparing different binary version with each other.
-// It's goal is to limit the startup up outdated binaries as well as having an easier way to ensure the software
-// is on the latest version from a UI/UX perspective
+// V is the versioning information for a binary
 type V struct {
```

Comment:
`Info` that way the full name is `version.Info`.

Reply:
Will do!